### PR TITLE
Prisma configuration

### PR DESCRIPTION
**Pull request by alan-ms**
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to <repo_name>?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the CONTRIBUTING.md doc.
- [x] I have added necessary documentation (if appropriate).
- [x] Any dependent changes have been merged and published in downstream modules.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: back-end/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+version: '3'
```
Comment (ciro-c) [inline back-end/docker-compose.yml:1]: Pode apagar esse arquivo. Já tem um na raiz do projeto

File: back-end/prisma/schema.prisma [hunk 2]
```diff
@@ -0,0 +1,91 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
```
Comment (ciro-c) [inline back-end/prisma/schema.prisma:10]: Alterar para fazer a conexão com as variáveis de ambiente já disponibilizado. Desse jeito não vai conectar

**Review by ciro-c — CHANGES_REQUESTED** [review]
Por favor alterar

File: back-end/prisma/schema.prisma (hunk 2 above)
Comment (alan-ms) [reply back-end/prisma/schema.prisma:10]: Vai ter que rodar dessa forma, pois é dessa maneira que o ORM funciona.

File: back-end/docker-compose.yml (hunk 1 above)
Comment (alan-ms) [reply back-end/docker-compose.yml:1]: Retirei

**Review by ciro-c — APPROVED** [review]

**Pull request merged**